pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -1052,24 +1052,31 @@ func getTaskrunWorkspaces(ctx context.Context, pr *v1beta1.PipelineRun, rpt *res
 	return workspaces, pipelinePVCWorkspaceName, nil
 }
 
-// taskWorkspaceByWorkspaceVolumeSource is returning the WorkspaceBinding with the TaskRun specified name.
-// If the volume source is a volumeClaimTemplate, the template is applied and passed to TaskRun as a persistentVolumeClaim
-func taskWorkspaceByWorkspaceVolumeSource(wb v1beta1.WorkspaceBinding, taskWorkspaceName string, pipelineTaskSubPath string, owner metav1.OwnerReference) v1beta1.WorkspaceBinding {
+// taskWorkspaceByWorkspaceVolumeSource returns the WorkspaceBinding to be bound to each TaskRun in the Pipeline Task.
+// If the PipelineRun WorkspaceBinding is a volumeClaimTemplate, the returned WorkspaceBinding references a PersistentVolumeClaim created for the PipelineRun WorkspaceBinding based on the OwnerReference.
+// Otherwise, the returned WorkspaceBinding references the same volume as the PipelineRun WorkspaceBinding
+// The returned WorkspaceBinding is combined with the pipelineTaskSubPath.
```

Comment:
I'm sort of confused what this means-- do you just mean the file paths are joined?

Reply:
Yes, I just meant the file paths are joined. I have updated the docstring for clarification, PTAL!